### PR TITLE
Add content to main page and each page

### LINKED ISSUE #58: Add explainer text at the top of each page
# Gender by culture

This plot shows the number of gendered biographies of men and women in Wikidata, by "Culture". <p><strong>culture </strong>is determined by using translating <em>ethnic group</em>, <em>place of birth</em>, and <em>citizenship </em>into 1 of 9 world cultures as per <a href="https://en.wikipedia.org/wiki/Inglehart%E2%80%93Welzel_cultural_map_of_the_world">Inglehart-Welzel map of the world</a> with Mechanical Turk. Then we take the consensus of the three aggregated variables. All aggregation maps are available for inspection <a href="https://github.com/notconfusing/WIGI/tree/master/helpers/aggregation_maps">on github</a>.</p> 

As of October 2014, only about 45% of biographies, had culture, so this data is incomplete.
# Gender by Country

This plot shows the the percentage of biographies of women by country of birth. Country is determined by seeing if the place of birth is a country, or if it is a city, see if the city has a country property in Wikidata.

The changes show the <em>change in total biographies</em>, and <em>change in percentage</em> since the previous snapshot date.

As of October 2014, only about 24% of biographies, had place of birth, so this data is incomplete.
# Gender by Date of Birth

This plot shows the Date of Birth (DoB) and Date of Death (DoD) of each biography in Wikidata, by gender. Non-binary genders, by last count there are 9 nonbinary genders, are displayed in the tables, and  accounted for in the <a href="http://wigi.wmflabs.org/snapshot_data/"> full data set</a>.

The changes show the <em>change in totals</em> since last snaphshot, for each year.  

As of October 2014, only about 58% and 28% of biographies, had date of birth and date of death data, respectively, so this data is incomplete.
# Gender by Language

This plot shows the top 40 Wikipedia Languages (by number of gendered biographies), and compares their number of gendered biographies to female percentage of those biographies. <strong>Hover to view the Language name.</strong>

The changes show the <em>number of biographies added</em> and the <em>female percentage change</em> between the two latest snapshots.

As of October 2014 about 99% of biographies were attached to at least one Wikipedia site, so this data is mostly complete.

### PR DESCRIPTION
**Pull request by raivivek**
Should close #58 and #51.

Signed-off-by: Vivek Rai vivekraiiitkgp@gmail.com

### CODE REVIEW

File: stories/wigi.md [hunk 1]
```diff
@@ -10,9 +10,20 @@
 .. template: mainpage.tmpl
 -->
 
-GENDER GAP THROUGH TIME AND SPACE : A JOURNEY THROUGH WIKIPEDIA BIOGRAPHIES
-===========================================================================
+<strong>Wi</strong>kipedia <strong>G</strong>ender Indicators (<strong>WIGI</strong>)
+-------------------------------------------------------------------------------------
 
-It is an unfortunate but unavoidable fact that encyclopedias always have had a gender bias. One of the dimensions of this bias is the contributors’ gender distribution (Thomas 1992; Reagle and Rhue 2011). Just as there were very few women authors among contributors to traditional, printed encyclopedias, recent surveys indicate that women constitute only around 13%-16% of Wikipedia contributors, or Wikipedians (Glott, Schmidt, & Ghosh 2010; Hill and Shaw 2013). 
+![36th_g8_summit_member_cropped_36th_g8_summit_member_20100625_1080](https://cloud.githubusercontent.com/assets/1494283/10463045/bfbe87f4-71a7-11e5-9851-59778a9f9ee0.jpg)
```
Comment (hargup) [inline stories/wigi.md]: Why do we have this photograph of bureaucrats here? I guess a better alternative will be the infographic Frances made.

Comment (raivivek) [inline stories/wigi.md]: Please comment on #51.

**Comment by raivivek** [comment]
@hargup Holding the photo discussion for a while, but I guess the rest can be merged?

**Comment by hargup** [comment]
Yeah everything else is fine, we can change the photo later.

**Pull request merged**